tickets/src/main/java/no/unit/nva/publication/ticket/update/UpdateTicketHandler.java

```diff
@@ -61,33 +60,24 @@ public class UpdateTicketHandler extends TicketHandler<UpdateTicketRequest, Void
         "User {} is not authorized to manage ticket {} for publication {}";
     private static final String TICKET_ASSIGNEE_UPDATE_MESSAGE =
         "User {} updates assignee to {} for publication {}";
+    public static final String UPDATE_FORBIDDEN_MESSAGE = "Updating ticket {} for publication {} is forbidden for user {}";
     private final TicketService ticketService;
     private final ResourceService resourceService;
     private final DoiClient doiClient;
-    private final UriRetriever uriRetriever;
```

Comment:
Not in use?

Reply:
Lack of PMD
Causes us to miss these things
Good to remove it